### ISSUE TITLE
[feature] Integration with Google Calendar?

### ISSUE THREAD

**Issue by justindra** [nicored/gotickyourself#11]
Just an idea, but any items on G Cal that involve other people can be automatically set to be used as meetings?

**Comment by nicored** [comment]
Yes, I actually thought about that, and I shall add it to the list
